di/src/main/java/com/example/di/KoinModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.di
+
+import com.example.data.service.WeatherService
+import com.example.domain.usecase.UseCaseInterfaceGetCity
+import org.koin.dsl.module
+import com.example.domain.service.WeatherInterfaceService
+import com.example.domain.usecase.implement.UseCaseGetCity
+
+val repositoriesModule = module {
+    factory<WeatherInterfaceService> { WeatherService() }
+}
+val useCasesModule = module {
+    factory<UseCaseInterfaceGetCity> { UseCaseGetCity() }
```

Comment:
both of these injectioons could be a single

Reply:
Done